controllers/borrowingsController.js

```diff
@@ -0,0 +1,133 @@
+import Borrowing from "../models/borrowings.model.js";
+import Book from "../models/books.model.js";
+import Member from "../models/members.model.js";
+
+// borrow book
+export async function borrowBook(req, res) {
+    let member = await Member.findAll({where: {member_id: req.body.member_id}});
+    let book = await Book.findAll({where: {book_id: req.body.book_id}});
+    let borrowing;
+
+    try {
+        if (member && book.no_of_copies > 0) {
+            req.body.returned = false;
+            borrowing = await Borrowing.create(req.body);
+            // decrement number of copies of a book
+        }
+
+        if (borrowing) {
+            res.status(200).json({
+                success: true,
+                message: "Book borrowed successfully",
+                data: borrowing
+            });
+        } else {
+            res.status(500).json({
+                success: false,
+                message: "Borrowing book failed"
+            });
+        }
+    } catch (err) {
+        console.log(err);
+        res.status(500).json({
+            success: false,
+            message: "Oops! Something went wrong."
+        });
+    }
+}
+
+// return book
+export async function returnBook(req, res) {
+    let book = await Book.findAll({where: {book_id: req.body.book_id}});
+    let member = await Member.findAll({where: {member_id: req.body.member_id}});
```

Comment:
Since you are looking for a single `User` and a single `Book` it is good to user `await Book.findOne({where: {book_id: req.body.book_id}});` and `await Member.findOne({where: {member_id: req.body.member_id}});`. This way you are sure you're getting the first entry matching your parameters.